components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -79,13 +79,21 @@ class P3Microphysics : public AtmosphereProcess
         // Exner
         const Spack opmid(pmid(icol,ipack));
         const Smask opmid_mask(!isnan(opmid) and opmid>0.0);
-        auto oexner = physics_fun::get_exner(opmid,opmid_mask);
+        auto oexner = PF::exner_function(opmid);
```

Comment:
While we're at it, can we avoid `oblah` names, where possible? I get that `pmid` already exists, so we can't call the pack `pmid` (although p_mid works), but for other stuff like exner, can we just call it `exner`? I find the `o` in front a bit confusing, since I keep thinking it stands for something, like 'output', or something.